inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -248,8 +248,18 @@ bool layout_optimizer::can_fuse_reorder_to_prev(program_node& prev, program_node
          fmt_next == format::b_fs_yx_fsv16 || fmt_next == format::b_fs_zyx_fsv16 || fmt_next == format::bs_fs_yx_bsv16_fsv16))
         return true;
 
-    if (prev.is_type<permute>())
-        return true;
+    if (prev.is_type<permute>() &&
+        ((fmt_prev == format::bfzyx && fmt_next == format::b_fs_zyx_fsv32) ||
```

Comment:
do we need explicit layouts list here? maybe `fmt_next.dimension() == fmt_prev.dimension()` will work?

Reply:
I think it will be working. Just to make it safe :D because those formats have corresponding fusing tests. But I think it will be okay, I will update it.

Reply:
Done